### PR TITLE
space: Clarify the return object of get_cell_list_contents

### PR DESCRIPTION
**Pull request by rht**

### CODE REVIEW

File: mesa/space.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 import itertools
 import math
+from warnings import warn
```
Comment (rht) [inline mesa/space.py:18]: This is unrelated refactor, accidentally comitted in. Oh well.

**Review by tpike3 — APPROVED** [review]
This is good I am not sure why we are import warn three different times in the same file.
